keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -881,6 +887,73 @@ Other components:
 * check how the change of meaning of resource requests influence other
   Kubernetes components.
 
+### Instrumentation
+
+The kubelet will record the following metrics:
+
+#### `kubelet_container_resize_requests_total` 
+
+This metric tracks the total number of resize requests observed by the Kubelet, counted at the container level.
+A single pod update changing multiple containers will be considered separate resize requests.
+
+Labels: 
```

Comment:
Should this be broken down by the result of the resize attempt?  E.g. admitted/infeasible/deferred?

Reply:
Counting the total of infeasible & deferred gets a little tricky, since we don't want to increment it every time the resize is retried. It might be sufficient to count changes to the resize conditions? I don't think it's worth adding generation tracking just for this.

Reply:
Yeah if we want to check that a particular deferred resize is getting retried, there are fields in the pod status we can check for that, but I'm not sure how useful it would be as a metric in aggregate since it really depends on the user's workloads